notes/health-care-evaluation.vbs

```diff
@@ -3997,360 +3952,606 @@ If HC_form_name = "No Form - Ex Parte Determination" Then
+
+				check_elig_renewal_date = DateAdd("d", 0, check_elig_renewal_date)
+				check_income_asset_renewal_date = DateAdd("d", 0, check_income_asset_renewal_date)
+				elig_renewal_date = DateAdd("d", 0, elig_renewal_date)
+				' income_asset_renewal_date = DateAdd("d", 0, income_asset_renewal_date)
+
+				' MsgBox "check_elig_renewal_date - " & check_elig_renewal_date & vbCr & "elig_renewal_date - " & elig_renewal_date
+				'Validate Elig Renewal Date to ensure it is set for 1 year from current Elig Renewal Date
+				If check_elig_renewal_date <> DateAdd("yyyy", 1, elig_renewal_date) THEN err_msg = err_msg & vbCr & "* The Elig Renewal Date should be set for 1 year from the current renewal month and year."
+
+				'Validate Income/Asset Renewal Date to ensure it is the same as the Elig Renewal Date or set for 6 months from original Elig Renewal Date for cases with a spenddown:
+				'TO DO - determine how to determine if meets spenddown?
+				' If check_income_asset_renewal_date <> DateAdd("Y", 1, income_asset_renewal_date) OR check_income_asset_renewal_date <> DateAdd("M", 6, income_asset_renewal_date) THEN
+				If check_income_asset_renewal_date <> check_elig_renewal_date Then err_msg = err_msg & vbCr & "* The Income/Asset Renewal Date should be be the same as the Elig Renewal Date. For cases with a spenddown that do not meet an exception listed in EPM 2.3.4.2 MA-ABD Renewals, enter a date six months from the original ELIG Renewal Date."
+				'TODO - put back the funcitonality for spenddowns
+
+				'Validate that Exempt from 6 Mo IR is set to N
+				If check_exempt_6_mo_ir_form <> "N" THEN err_msg = err_msg & vbCr & "* You must enter 'N' for Exempt from 6 Mo IR."
+
+				'Validate that ExParte field updated to Y
+				If check_HC_ex_parte_determination <> "Y" THEN err_msg = err_msg & vbCr & "* You must enter 'Y' for ExParte."
+
+				'Validate that ExParte Renewal Month is correct
+				'TO DO - add validation to ensure that date updated in HC renewal screen is the same as date provided in SQL table
+				If check_ex_parte_renewal_month_year = "__ ____" THEN err_msg = err_msg & vbCr & "* You must enter the month and year for the Ex Parte renewal month."
+
+				'Error message handling
+				IF err_msg <> "" THEN MsgBox "*** NOTICE!***" & vbNewLine & err_msg & vbNewLine
+			Loop until err_msg = ""
+				'Add to all dialogs where you need to work within BLUEZONE
+				CALL check_for_password(are_we_passworded_out)			'function that checks to ensure that the user has not passworded out of MAXIS, allows user to password back into MAXIS
+		LOOP UNTIL are_we_passworded_out = false					'loops until user passwords back in
+
+
+
+		'For ex parte denial, write information to case note
+		CALL write_variable_in_case_note("*** EX PARTE DETERMINATION - EX PARTE DENIED ***")
+		CALL write_variable_in_case_note("Phase 2 - The case has been evaluated for ex parte and has been denied based on the information provided.")
```

Comment:
Need detail in the CASE/NOTE about when the renewal is rescheduled for.

Reply:
Updated to include CASE/NOTE about the updated HC renewal date